src/useq/_mda_event.py

```diff
@@ -115,6 +115,8 @@ class MDAEvent(UseqModel):
         The action to perform for this event.  By default, [`useq.AcquireImage`][].
         Example of another action is [`useq.HardwareAutofocus`][] which could be used
         to perform a hardware autofocus.
+    keep_shutter_open : bool
+        Keep the shutter open between during this event. By default, `False`.
```

Comment:
what does "between during this event" mean?  Does it mean "Do not close the illumination shutter at the end of this event?"

Reply:
it's a mistake, it should be updated as:

`If True, the shutter should be kept open during this event. By default, False.`